### PR TITLE
HG-846 More work on top-bar component in mercury

### PR DESCRIPTION
**Pull request by lizlux**
This is a follow up PR to https://github.com/Wikia/mercury/pull/1358, addressing leftover feedback and fixing bugs as I see them. 

Changed: 
* fixing fast click issue
* updating style guide version
* using partial for web-components

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -5,25 +5,25 @@
   "license": "MIT",
   "private": true,
   "dependencies": {
+    "cropper": "~0.11.0",
```
Comment (lizlux) [inline bower.json]: Looks like the latest version of bower will alphabetize your existing dependencies when running `bower install foo --save`

File: server/views/_layouts/ember-main.hbs [hunk 2]
```diff
@@ -15,9 +15,7 @@
 			<meta name="fragment" content="!">
 		{{/if}}
 
-		<!-- Web components and library import -->
-		<script> window.Polymer = window.Polymer || {}; window.Polymer.dom = 'shadow'; </script>
-		<link rel="import" href="/front/vendor/wikia-style-guide/dist/components/import-built.html" async>
+		{{> web-components}}
```
Comment (lizlux) [inline server/views/_layouts/ember-main.hbs]: @rogatty

Comment (rogatty) [inline server/views/_layouts/ember-main.hbs]: Cool, thanks!

**Comment by rogatty** [comment]
LGTM

File: server/views/_layouts/ember-main.hbs (hunk 2 above)
Comment (lizlux) [inline server/views/_layouts/ember-main.hbs]: thank you for the suggestion!

**Pull request merged**